### PR TITLE
[Doc] Document `<EditInDialogButton>` deletion side effect

### PR DESCRIPTION
**Pull request by fzaninotto**
- [x] Add "Redirection After Deletion" section
- [x] Detail missing props

### CODE REVIEW

File: docs/EditInDialogButton.md [hunk 1]
```diff
@@ -51,24 +61,297 @@ const CompanyShow = () => (
...
+        </SimpleForm>
+    </Edit>
+);
+```
+{% endraw %}
 
 ## Combining With `<CreateInDialogButton>`
 
-Below is an example of an `<Edit>` view, inside which is a nested `<Datagrid>`, offering the ability to **create**, **edit** and **show** the rows thanks to [`<CreateInDialogButton>`](./CreateInDialogButton.md), `<EditInDialogButton>` and [`<ShowInDialogButton>`](./ShowInDialogButton.md):
+Below is an example of an `<Edit>` view, inside which is a nested `<Datagrid>`, offering the ability to **create**, **edit** and **show** the rows thanks to [`<CreateInDialogButton>`](./CreateInDialogButton.md), `<EditInDialogButton>` and [`<EditInDialogButton>`](./EditInDialogButton.md):
```
Comment (slax57) [inline docs/EditInDialogButton.md:354]: Not sure this replacement is intentional. You should keep `ShowInDialogButton` here IMO.

File: docs/EditInDialogButton.md [hunk 2]
```diff
@@ -82,15 +365,15 @@ import {
   SelectField,
   SelectInput,
   SimpleForm,
-  SimpleShowLayout,
+  SimpleForm,
```
Comment (slax57) [inline docs/EditInDialogButton.md:368]: same

File: docs/EditInDialogButton.md [hunk 3]
```diff
@@ -82,15 +365,15 @@ import {
...
   TextField,
   TextInput,
   useRecordContext,
 } from "react-admin";
 import {
   CreateInDialogButton,
   EditInDialogButton,
-  ShowInDialogButton,
+  EditInDialogButton,
```
Comment (slax57) [inline docs/EditInDialogButton.md:376]: same

File: docs/EditInDialogButton.md [hunk 4]
```diff
@@ -108,12 +391,12 @@ const CustomerForm = (props) => (
 );
 
 const CustomerLayout = (props) => (
-  <SimpleShowLayout {...props}>
+  <SimpleForm {...props}>
```
Comment (slax57) [inline docs/EditInDialogButton.md:394]: same

File: docs/EditInDialogButton.md [hunk 5]
```diff
@@ -140,10 +423,10 @@ const NestedCustomersDatagrid = () => {
     </EditInDialogButton>
   );
 
-  const showButton = (
-    <ShowInDialogButton fullWidth maxWidth="md">
+  const EditButton = (
```
Comment (slax57) [inline docs/EditInDialogButton.md:426]: same

File: docs/EditInDialogButton.md [hunk 6]
```diff
@@ -161,7 +444,7 @@ const NestedCustomersDatagrid = () => {
         {/* Using a component as label is a trick to render it in the Datagrid header */}
         <DatagridActionsColumn label={createButton}>
           {editButton}
-          {showButton}
+          {EditButton}
```
Comment (slax57) [inline docs/EditInDialogButton.md:447]: ...and same
(I think you got the idea)

**Review by slax57 — APPROVED** [review]

File: docs/EditInDialogButton.md [hunk 7]
```diff
@@ -51,20 +61,293 @@ const CompanyShow = () => (
...
 
-It accepts the following props:
+## Props
+
+`<EditInDialogButton>` accepts the following props:
+
+| Prop           | Required | Type              | Default | Description |
+| -------------- | -------- | ----------------- | ------- | ----------- |
+| `children`     | Required | `ReactNode`       |         | The content of the dialog. |
+| `ButtonProps`  | Optional | `object`          |         | Object containing props to pass to Material UI's `<Button>`. |
```
Comment (adguernier) [inline docs/EditInDialogButton.md:76]:
```suggestion
| `ButtonProps`  | Optional | `object`          |         | Object containing props to pass to Material UI's [`<Button>`](https://mui.com/material-ui/react-button/). |
```

**Review by adguernier — APPROVED** [review]

**Pull request merged**